app/src/pages/IndividualRequest.tsx

```diff
@@ -152,8 +173,8 @@ const IndividualRequest = () => {
     }}/>
     : <>
         <h1>You do not have access to this record.</h1>
-        <p>If you think you are seeing this by mistake, contact your administrator.</p>
-        <LinkButton link="/" style={buttonStyles.secondary}>Back</LinkButton>
+        <p style={{ margin: '1em 0'}}>If you think you are seeing this by mistake, contact your administrator.</p>
```

Comment:
Oof, inline styling 🙅🏼

Reply:
What would you recommend instead?

Reply:
CSS utility class for applying: `margin-block: 1em`

Reply:
Made an update using a separate CSSProperties object.